packages/langium/src/workspace/workspace-manager.ts

```diff
@@ -30,7 +30,7 @@ export interface WorkspaceManager {
 
 }
 
-export class DefaultWorkspaceManager {
+export class DefaultWorkspaceManager implements WorkspaceManager {
```

Comment:
Interesting that nobody complained before :)

Reply:
This is actually optional in TypeScript because types are compared structurally. This is mostly useful to check that the interface methods are implemented correctly.